data/etc/apache2/conf-available/tahoe-plinth.conf

```diff
@@ -0,0 +1,11 @@
+# Tahoe-LAFS Storage Node web interface
+
+# See if Tahoe-LAFS has basic auth
+# Enable Tahoe only from localhost
+# Do basic auth and LDAP from apache reverse proxy
```

Comment:
Are these TODOs? I think we will want to have login required at least (through apache)